guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@dgmstuart `git pull` causes merge commits in some circumstances. When `--no-ff` is set it always creates merge commits when master != origin/master.

But when that option isn't set, it probably won't happen very often, if everybody is doing everything else correctly. But I would still advocate using `pull --ff-only`. This would avoid situations where somebody's committed something to master they didn't meant to be on the master branch, or where something has otherwise gone wrong.